File: DownloadFiles/TGNet/Chapter05/C#/Exercise5-3/AutoTesterwithnewForm/AutoTester/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using System.Diagnostics;

namespace AutoTester
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
       
        private void btnStartApp_Click(object sender, EventArgs e)
        {
        
            try
            {
                TestLib.Logfile = "c:\\testlog.log";
                TestLib.StartProgram(txtAppName.Text);
            }
            catch (Exception ex)
            {
                MessageBox.Show(ex.Message.ToString());
            }
        }

        private void exitToolStripMenuItem_Click(object sender, EventArgs e)
        {
            this.Close();
        }

        private void aboutToolStripMenuItem_Click(object sender, EventArgs e)
        {
            AboutBox1 aForm = new AboutBox1();
            aForm.ShowDialog();
        }

        private void startApplicationToolStripMenuItem_Click(object sender, EventArgs e)
        {
            btnStartApp.PerformClick();
        }

      }
    }

File: DownloadFiles/TGNet/Chapter05/VB/Exercise5-2/AutoTesterwithSharedClass/Chapter5Demo/Form1.vb
Public Class Form1
    

    Private Sub Button1_Click(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles btnStartApp.Click
        Try
            TestLib.Logfile = "c:\testlog.log"
            TestLib.StartProgram(txtAppname.Text)
        Catch ex As Exception
            MessageBox.Show(ex.Message.ToString)
        End Try
    End Sub
End Class
